src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TodoList from '../TodoList';
+import './TodoForm.css';
+
+export class TodoForm extends Component {
+  state = {
+    todos: [...this.props.todos],
+    name: '',
+    userId: 0,
+    title: '',
+    maxTitleLength: 40,
```

Comment:
if you want to save maxTitleLength just create const, no need to save it in state